site/en/install/gpu.md

```diff
@@ -98,15 +98,15 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
 # Install development and runtime libraries (~4GB)
 <code class="devsite-terminal">sudo apt-get install --no-install-recommends \
     cuda-10-0 \
-    libcudnn7=7.4.1.5-1+cuda10.0  \
-    libcudnn7-dev=7.4.1.5-1+cuda10.0
+    libcudnn7=7.6.0.64-1+cuda10.0  \
+    libcudnn7-dev=7.6.0.64-1+cuda10.0
 </code>
 
 # Install TensorRT. Requires that libcudnn7 is installed above.
 <code class="devsite-terminal">sudo apt-get update && \
         sudo apt-get install nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
```

Comment:
Ok, last item.  We need to test if the install of the libnvinfra can be done without needing to install the nvinfer-runti-trt-repo.  I suspect Yes given the code below is install a version different than the repo; but I would like that validated.  Easy to do if you have a local fresh ubuntu docker or VM.  I may get to it but I suspect anyone would be faster than me.